src/scanpy/plotting/_stacked_violin.py

```diff
@@ -814,6 +815,7 @@ def stacked_violin(
         var_group_labels=var_group_labels,
         var_group_rotation=var_group_rotation,
         layer=layer,
+        categories_order=order,
```

Comment:
What prompted this?

Reply:
just a bug I found, I saw that `order` was unused.

Reply:
Ok, do you think it needs a test? Some way to check this? Perhaps overkill, though....up to you

Reply:
I changed my mind. `categories_order` is the standard name everywhere else so I just removed the unused `order` instead.